### DISCUSSION TITLE
toast-uic hart tooltip questions

### DISCUSSION THREAD

**Discussion by jewon7377** [nhn/tui.chart#744] [Q&A]
Hello

In the image below,
You are attempting to change the text color on the body part of the tooltip, but it is not being applied.
I'm asking if there's anything wrong.

source part)
  theme: {
      legend: {
	  label: {
					 fontSize: `12,`
					 fontFamily: 'Lato'
				 }
			},
			yAxis: {
				title: '',
				label: {
					fontFamily: 'Lato',
					fontSize: 12,
					formatter: (value) => `${value.toString().replace(/\B(?<!\.\d*)(?=(\d{3})+(?!\d))/g, ',')} 원`
				},
			},
			xAxis: {
				title: '',
				label: {
					fontFamily: 'Lato',
					fontSize: 12
				}
			},
			**_tooltip: {
				body: {
					fontSize: 100,
					color: 'white',
				},
			},_**
		},
``

![example](https://user-images.githubusercontent.com/33106890/168251522-23c316c1-1754-4766-a772-b6c929787030.png)

**Comment by jwlee1108** [comment]
@jewon7377 

Hello, Can I change the color value to Hex instead of a symbol? 
`white` → `#FFFFFF `

![image](https://user-images.githubusercontent.com/30035674/169176653-76f8fb90-bf00-476a-81ec-fd8063d87bd5.png)
